static/src/js/containers/GranuleResultsBodyContainer/__tests__/GranuleResultsBodyContainer.test.js

```diff
@@ -78,7 +78,7 @@ describe('mapDispatchToProps', () => {
     expect(spy).toBeCalledWith('granuleId')
   })
 
-  test('onGenerateNotebook calls actions.excludeGranule', () => {
+  test('onGenerateNotebook calls actions.generateNotebook', () => {
     const dispatch = jest.fn()
     const spy = jest.spyOn(actions, 'excludeGranule')
```

Comment:
This should also be generateNotebook.

Reply:
🤦 Nice catch. Looks like we were not running `clearAllMocks` so the spy from the previous test was hiding the issue. Added a `clearAllMocks` in a `beforeEach` to make sure that doesn't happen again.